docs/v0.5/runtimes/streaming.md

```diff
@@ -116,3 +103,49 @@ riff streaming processor create time-averager \
   --input in \
   --output out
 ```
+
+## Testing with dev-utils
+
+The [dev-utils](https://github.com/projectriff/dev-utils/) container offers CLI utilities which can be used to publish messages to an input stream and subscribe to messages on an output stream.
+
+To run dev-utils in a pod in the `default` namespace, using a service account called `dev-utils`:
```

Comment:
please use the install instructions recommended in the dev-utils readme

Reply:
@scothis those instructions introduce yet another name for users to understand.
I'd prefer to use the name 'dev-utils' for the pod and the service account.
Happy to submit a PR for the dev-utils README.

Reply:
using a riff- prefix identifies the resource better, dev-utils is very generic

Reply:
as @trisberg points out, the name `dev-utils` is very generic and it won't be obvious looking at the pod what it is. The name change in dev-utils repo was intentional.

Reply:
ok